documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -1381,9 +1381,9 @@ a|_n/a_
 |`FLOAT32`
 a|_n/a_
 
-|`FLOAT[(M,D)]`
-|`FLOAT64`
-a|_n/a_
+|`FLOAT(P)` or `FLOAT[(M,D)]`
+|`FLOAT32` or `FLOAT64`
+a|The precision `P` from 0 to 23 results to `FLOAT32`, from 24 to 53 results to `FLOAT64`. As of MySQL 8.0.17, the nonstandard FLOAT(M,D) and DOUBLE(M,D) syntax is deprecated, and should expect support for it be removed in a future version of MySQL, so set to `FLOAT64` as default.
```

Comment:
Could you please take a look at https://debezium.io/documentation/reference/1.9/connectors/oracle.html#oracle-numeric-types and format/descibe it in a similar way so it is consistent accross different connectors?

Reply:
done